aiogram/types/chat.py

```diff
@@ -11,6 +11,7 @@
 from .chat_permissions import ChatPermissions
 from .chat_photo import ChatPhoto
 from .input_file import InputFile
+from .message import Message
```

Comment:
It's a bad practice for type cross importing. Sometimes it may produce a cycled import. String typing is better solution